include/cantera/thermo/ThermoFactory.h

```diff
@@ -12,6 +12,7 @@
 
 #include "ThermoPhase.h"
 #include "cantera/base/FactoryBase.h"
+#include "cantera/base/stringUtils.h"
```

Comment:
I think this `include` can be moved to the `.cpp` file, to avoid dragging it in everywhere `ThermoFactory.h` is included.

Reply:
Absolutely. I had this included when I tested the templated version.